README.md

```
@@ -134,12 +134,14 @@ redis-server --port 6379
 
 ### Install Python dependencies
 
-quipucords uses uv to manage Python dependencies. To install or update them, simply run:
+quipucords uses uv to manage Python dependencies. To install or update them, run:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)